app/config.py

```diff
@@ -318,8 +318,8 @@ class Config(object):
             },
             'send-scheduled-comp-and-pen-sms': {
                 'task': 'send-scheduled-comp-and-pen-sms',
-                # Every 2 minutes past every hour from 13 through 21 on every day-of-month from 22 through end-of-month
-                'schedule': crontab(hour='13-21', day_of_month='22-31', minute='*/2'),
+                # Every 1 minute past every hour from 13 through 21 on every day-of-month from 22 through end-of-month
+                'schedule': crontab(hour='13-21', day_of_month='22-31', minute='*/1'),
```

Comment:
I believe the minute parameter is no longer necessary in this case, but we can leave it in if we want to be explicit.

Reply:
I can never remember all their rules 😆